src/Network/Network/AzureFirewallPolicy/NatRule/NewAzureFirewallPolicyNatRuleCollectionCommand.cs

```diff
@@ -53,18 +53,6 @@ public class NewAzureFirewallPolicyNatRuleCollectionsCommand : NetworkBaseCmdlet
             IgnoreCase = false)]
         public string ActionType { get; set; }
 
-
-        [Parameter(
```

Comment:
removing mandatory parameter is breaking change

Reply:
I understand this is a breaking change but we are fine with it because Firewall Policy is still a preview product. Please accept this change.